backend/api/profile.js

```diff
@@ -0,0 +1,28 @@
+const handler = require('../handler/handler');
+const Host = require('../handler/Classes/Host');
+
+const profile = (req, res) => {
+    let response;
+    // eslint-disable-next-line no-console
+    console.log('INFO: /profile has been called.');
+
+    let data = handler.getData();
+    
+    if (req.body.myToken) {
+        let myHost = Host.getHostByToken(req.body.myToken, data.hosts);
+        if (myHost) {
+            response = {error: false, host: myHost.getData()};
+            res.status(200);
+        } else {
+            response = {error: true, message: 'Authorization failed. No or expired token.'};
+            res.status(400);
+        }    
+    } else {
+        response = {error: true, message: 'Authorization failed. No or expired token.'};
+        res.status(400);
```

Comment:
please use more descriptive status codes. in this case, it would be HTTP 401 Unauthorized. Implement this for the rest of the API routes too, please. 

https://en.wikipedia.org/wiki/List_of_HTTP_status_codes#4xx_client_errors